server/gql/models/tests/students/query.test.js

```diff
@@ -1,22 +1,36 @@
 import get from 'lodash/get';
 import { getResponse, mockDBClient, resetAndMockDB } from '@utils/testUtils';
+import { subjectsTable } from '@server/utils/testUtils/mockData';
 
 describe('Student graphQL-server-DB query tests', () => {
-  const id = 1;
+  const studentId = 1;
   const studentOne = `
     query {
-      student (id: ${id}) {
+      student (id: ${studentId}) {
         id
         name
+        subjects {
+          edges {
+            node {
+              id
+            }
+          }
+        }
       }
     }
   `;
-  it('should request for students', async () => {
+  it('should request for subjects related to the students', async () => {
     const dbClient = mockDBClient();
     resetAndMockDB(null, {}, dbClient);
 
+    jest.spyOn(dbClient.models.subjects, 'findAll').mockImplementation(() => [subjectsTable[0]]);
+
     await getResponse(studentOne).then(response => {
       expect(get(response, 'body.data.student')).toBeTruthy();
+
```

Comment:
let's test against the shapee of the object

Reply:
I did this test case in the next PR, is it okay?
That test includes limit and offset scenario also.

Reply:
fair